app/src/main/java/com/bc/gordiansigner/service/AccountService.kt

```diff
@@ -45,20 +47,20 @@ class AccountService @Inject constructor(
         }.subscribeOn(Schedulers.computation())
     }
 
-    fun getHDKey(fingerprint: String) =
-        getHDKeys()
-            .map { keys -> keys.first { it.fingerprintHex == fingerprint } }
-            .flatMap { updateKeyInfoLastUsed(it.fingerprintHex).andThen(Single.just(it)) }
+    fun getSeed(fingerprint: String, chain: Network = Network.TEST) =
```

Comment:
Should unify the word `chain` and `network`. For declared enum `Network`, i would recommend use `network` in the app.